UnityProject/Assets/Prefabs/Items/Implants/Organs/HumanHead.prefab

```diff
@@ -375,6 +380,11 @@ PrefabInstance:
       propertyPath: m_LocalEulerAnglesHint.z
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 7822575272286095514, guid: e241039a7933eb347bd1a9bb67a9078a,
+        type: 3}
+      propertyPath: deadlyDamageInOneHit
+      value: 100
```

Comment:
this doesn't fix the issue with explosions, this just hides it.
Smaller explosions shouldn't be doing over 30 damage

Reply:
now that you mention it. We gave walls that kind of health because we didn't have damage deflection and people kept punching walls until they fell down. Maybe we need to go back and re-evaluate those values.

Reply:
It still means that you need to do like a hundred damage on tile , To break the wall, and that's too high in terms of health